python/tests/spark/test_functions.py

```diff
@@ -139,6 +139,7 @@ def test_video_to_images(spark: SparkSession):
     """Test extract video frames from YouTubeVideo/VideoStream types
     into list of Image assets.
     """
+    youtube_uri = os.path.join(os.path.dirname(__file__), "..", "assets")
```

Comment:
this is going to pollute the repo directories. with pytest you can pass in `tmp_dir` into your test function and start using it.  https://docs.pytest.org/en/stable/tmpdir.html

Reply:
got it 👍